### PR TITLE
Chat message management

### PR DESCRIPTION
**Pull request by esmaoui**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatManageMessageTestIT.java [hunk 1]
```diff
@@ -223,4 +238,77 @@ public void test04_saveMeetingNote() {
...
+    String email = username + "@test.com";
+    String I;
+
+    navigationToolbar.goToAddUser();
+    userAddManagement.addUser(username, password, email, FirstName, LastName);
+    homePagePlatform.goToChat();
+    switchTo().window(1);
+    roomManagement.addRoom(room, username);
+    chatManagement.sendMessageInRoomOrSpace(room,message);
+    I=$(byClassName("message-time")).waitUntil(Condition.appear, Configuration.timeout).getText();
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatManageMessageTestIT.java]: 1- Variable must have meaningful name.
2- you must specify which message you will get its time using $(byText(message)).parent()....

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatMessageManagementTestIT.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        homePagePlatform.goToChat();
+        switchTo().window(1);
+        roomManagement.addRoom(room);
+        chatManagement.sendMessageInRoomOrSpace(room,message);
+        messageManagement.quoteMessage(message,DATA_NAME_USER1);
+        roomManagement.deleteRomm(room);
+    }
+
+    @Test
+    public void test05_SaveNotes(){
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatMessageManagementTestIT.java]: This test fails

File: chat/src/main/java/org/exoplatform/platform/qa/ui/chat/pageobject/MessageManagement.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    private final TestBase testBase;
+
+    public MessageManagement(TestBase testBase) {
+        this.testBase = testBase;
+    }
+
+    public void checkMessageMenu(String message){
+
+        ELEMENT_CHAT_LIST_MSG.find(byText(message)).hover();
+        ELEMENT_CHAT_LIST_MSG.find(byClassName("uiIconDots")).click();
```
Comment (ilyeskallala) [inline chat/src/main/java/org/exoplatform/platform/qa/ui/chat/pageobject/MessageManagement.java]: you must specify which message you want to open its drop down menu

**Review by ilyeskallala — CHANGES_REQUESTED** [review]

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatManageMessageTestIT.java (hunk 1 above)
Comment (esmaoui) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatManageMessageTestIT.java]: We find only a message time after add a room and send a message

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatMessageManagementTestIT.java (hunk 2 above)
Comment (esmaoui) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatMessageManagementTestIT.java]: done

File: chat/src/main/java/org/exoplatform/platform/qa/ui/chat/pageobject/MessageManagement.java (hunk 3 above)
Comment (esmaoui) [reply chat/src/main/java/org/exoplatform/platform/qa/ui/chat/pageobject/MessageManagement.java]: done

**Review by ilyeskallala — CHANGES_REQUESTED** [review]
Compilation error in project

**Pull request closed without merging**